src/storage/mod.rs

```diff
@@ -1671,7 +1696,10 @@ impl<E: Engine, L: LockManager> Storage<E, L> {
                     .get(priority_tag)
                     .inc();
 
-                // TODO: check_api_version
+                let keys = Some(&start_key)
+                    .into_iter()
+                    .chain(end_key.as_ref().into_iter());
```

Comment:
Why a "None" end_key (means "unbounded" I think) should also be checked ?

Besides, under current `check_api_version()` for Api V2 storage, an empty key will always be illegal, as it is of type "KeyPrefix::Unknown".

Reply:
It's because the unbounded range is prohibited in APIV2. Imaging a raw_delete_range request which ranges from "r\0k1" to "", if this is allowed, all transaction keys with "t{keyspaceid}" prefix will also be deleted.

Reply:
Got it.
Then I think it would be better to explicitly reject unbounded range and return corresponding error info.